src/cms/package.json

```diff
@@ -30,14 +31,27 @@
     "strapi-plugin-graphql": "^3.6.1",
     "strapi-plugin-upload": "3.6.1",
     "strapi-plugin-users-permissions": "3.6.1",
-    "strapi-provider-upload-aws-s3": "^3.6.8",
     "strapi-provider-email-nodemailer": "^3.6.8",
+    "strapi-provider-upload-aws-s3": "^3.6.8",
     "strapi-utils": "3.6.1",
     "utf8": "^3.0.0"
   },
   "engines": {
     "node": ">=10.16.0 <=14.x.x",
     "npm": "^6.0.0"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
I believe the license should be Apache 2.0

Reply:
Removed in https://github.com/bcgov/bcparks.ca/pull/229/commits/d275366bc0a67f346c6667e4b9e622cb20677b3c.  That's leftover from the previous team.  I just removed it since we already have a `LICENSE` document in the repo root.